datahub-web-react/src/app/identity/user/ViewResetTokenModal.tsx

```diff
@@ -88,14 +116,7 @@ export default function ViewResetTokenModal({ visible, userUrn, username, onClos
                 </ModalSectionParagraph>
                 <CreateResetTokenButton
                     onClick={() => {
-                        createNativeUserResetToken({
-                            variables: {
-                                input: {
-                                    userUrn,
-                                },
-                            },
-                        });
-                        setHasGeneratedResetToken(true);
+                        createNativeUserResetToken();
```

Comment:
nit: can just be `onClick={createNativeUserResetToken}`

Reply:
Good call! Fixed